front/src/common/Map/Settings/MapSettingsSpeedLimits.tsx

```diff
@@ -51,18 +52,20 @@ const FormatSwitch = ({ name, icon: IconComponent, color = '', disabled }: Forma
     );
   };
 
-  const dispatchSetSpeedLimitsTags = (item?: string) => {
-    if (item)
+  const dispatchSetSpeedLimitsTags = (item?: string | null) => {
```

Comment:
Can you change in SelectImproveSNCF the type of onChange => `onChange: (value: T) => void;` and then you can change the signature of this function:

```suggestion
  const dispatchSetSpeedLimitsTags = (item: string) => {
```